fiberzerolog/zerolog_test.go

```diff
@@ -181,15 +222,25 @@ func Test_Logger_All(t *testing.T) {
 		"protocol":      "http",
 		"pid":           float64(os.Getpid()),
 		"queryParams":   "foo=bar",
-		"resBody":       "Cannot GET /",
+		"resBody":       "Not Found",
 		"bytesReceived": float64(0),
-		"bytesSent":     float64(12),
+		"bytesSent":     float64(9),
 	}
 
 	var logs map[string]any
 	_ = json.Unmarshal(buf.Bytes(), &logs)
 
-	utils.AssertEqual(t, expected, logs)
+	for key, value := range expected {
+		utils.AssertEqual(t, value, logs[key])
+	}
+
+	latencyStr, ok := logs[FieldLatency].(string)
+	if !ok {
+		t.Errorf("Failed to parse latency from logs")
```

Comment:
use utils.AssertEqual instead

Reply:
We can't use AssertEqual because latency always varies and we can't have an equal value for AssertEqual

Reply:
@efectn We can only check the type for this field.

Reply:
> @efectn We can only check the type for this field.

Isn't ok boolean? We can check it's true

Reply:
OK came from interface type assertion, not the latency return itself, we can't AssertEqual that

Reply:
> OK came from interface type assertion, not the latency return itself, we can't AssertEqual that

I know, i just mean something like
```go
utils.AssertEqual(t, true, ok)
utils.AssertEqual(t, true, strings.Contains(latencyStr, "ms"))
```

Reply:
> > OK came from interface type assertion, not the latency return itself, we can't AssertEqual that
> 
> I know, i just mean something like
> 
> ```go
> utils.AssertEqual(t, true, ok)
> utils.AssertEqual(t, true, strings.Contains(latencyStr, "ms"))
> ```

Okay, that works